src/status_im/contexts/chat/messenger/composer/mentions/view.cljs

```diff
@@ -35,10 +35,9 @@
          (set-suggestions-state suggestions)
          (js/setTimeout #(set-suggestions-state suggestions) 300))
        (reanimated/animate opacity (if suggestions? 1 0)))
-     [suggestions?])
-    [reanimated/view
-     {:style (style/container opacity top theme)}
-     [rn/flat-list
+     [suggestions])
+    [reanimated/view {:style (style/container opacity top theme)}
+     [reanimated/flat-list
```

Comment:
> What do we gain by using a flat list here instead of the macro for? I checked with a simple for and it all works the same apparently, including the animation.

@ilmotta A flat-list is lazy, it's built to render a lot of items, it just renders what you see (window-size) and some windows above and below (it's configurable). Having a `for` inside a `scroll-view` (to keep the scrolling) will render all possible suggestions at once. Not the best for this use case.

Reply:
The list of mentions is limited by 10 items I think.

Reply:
In which case the flatlist is unnecessary. https://github.com/status-im/status-mobile/pull/21131

Reply:
Cool, then we can use a scrollview! I'll change it.